cdap-data-fabric/src/main/java/co/cask/cdap/data2/datafabric/dataset/RemoteDatasetFramework.java

```diff
@@ -199,6 +203,7 @@ private void addModule(Id.DatasetModule moduleId, Class<?> typeClass) throws Dat
     try {
       File tempFile = File.createTempFile(typeClass.getName(), ".jar");
       try {
+        // TODO: should be namespaced?
```

Comment:
no, I don't believe so.  It doesn't specify moduleType in the path, so shouldn't need namespace either, right